libs/common/src/test-helpers/generate-mock-audit-database-record.ts

```diff
@@ -0,0 +1,55 @@
+import { ObjectId } from 'mongodb';
+import {
+  generateTfmUserAuditDatabaseRecord,
+  generatePortalUserAuditDatabaseRecord,
+  generateSystemAuditDatabaseRecord,
+  generateNoUserLoggedInAuditDatabaseRecord,
+} from '../helpers/change-stream/generate-audit-database-record';
+
+export const generateMockTfmUserAuditDatabaseRecord = (mockUserId: string | ObjectId) => ({
+  ...generateTfmUserAuditDatabaseRecord(mockUserId),
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  lastUpdatedAt: expect.any(String),
+});
+
+export const generateMockPortalUserAuditDatabaseRecord = (mockUserId: string | ObjectId) => ({
+  ...generatePortalUserAuditDatabaseRecord(mockUserId),
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  lastUpdatedAt: expect.any(String),
+});
+
+export const generateMockSystemAuditDatabaseRecord = () => ({
+  ...generateSystemAuditDatabaseRecord(),
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  lastUpdatedAt: expect.any(String),
+});
+
+export const generateMockNoUserLoggedInAuditDatabaseRecord = () => ({
+  ...generateNoUserLoggedInAuditDatabaseRecord(),
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  lastUpdatedAt: expect.any(String),
+});
+
+/**
+ * @param mockUserId - mock user id
```

Comment:
I felt this function needed some documentation to explain the slightly odd `JSON.parse` stuff. It might be that a better function name would help but I couldn't decide on anything better